src/clientHandler.ts

```diff
@@ -0,0 +1,252 @@
+import * as vscode from 'vscode';
+import ShortUniqueId from 'short-unique-id';
+import {
+	Executable,
+	LanguageClient,
+	LanguageClientOptions,
+	ServerOptions,
+	State,
+	DocumentSelector
+} from 'vscode-languageclient/node';
+import {
+	config,
+	getFolderName,
+	getWorkspaceFolder,
+	normalizeFolderName,
+	sortedWorkspaceFolders
+} from './vscodeUtils';
+import * as path from 'path';
+import TelemetryReporter from 'vscode-extension-telemetry';
+
+export interface TerraformLanguageClient {
+	commandPrefix: string,
+	client: LanguageClient
+}
+
+let clients: Map<string, TerraformLanguageClient> = new Map();
```

Comment:
```suggestion
const clients: Map<string, TerraformLanguageClient> = new Map();
```
It is common to use const when the variable is never reassigned. The project contains a config for eslint that would also error for this (it is installed as a dependency but not run in scripts).
You can run it via `./node_modules/.bin/eslint .` or via `yarn eslint .` (if you have yarn installed – as yarn falls back to the `.bin` directory where npm complains instead).

Reply:
Thanks for pointing it out - I wasn't even aware we had a linter set up.

I already installed VS Code ESLint extension and fixed all the errors and any warnings in the new code. I also added a check to CI.